contributor_docs/inline_documentation.md

```diff
@@ -193,18 +193,21 @@ arc(50, 55, 80, 80, PI+QUARTER_PI, TWO_PI);
 </div>
 ```
 
-You can have multiple examples for one function, just make sure each has it's own `<div>` wrapping. 
+You can have multiple examples for one function, just make sure each example has it's own `@example` block seperated by a single empty line.
+
 ```
 @example
 <div>
 <code>arc(50, 50, 80, 80, 0, PI+QUARTER_PI, OPEN);</code>
 </div>
+
+@example
```

Comment:
there should be only one @example tag, then the html code for as many examples as you'd like

Reply:
Ok, i will remove it.